server.py

```diff
@@ -0,0 +1,114 @@
+from flask import Flask,jsonify, request
+import os
+import shutil
+import pathlib
+
+
+app = Flask(__name__)
+
+#==============================================heartbeat<3=========================
+
+@app.route('/heartbeat' , methods=['GET', 'POST'])
+def heartbeat():
+    return "204"
+
+#==============================================reading file=========================
+@app.route('/GET/<string:pathToFile>', methods=['GET'])
```

Comment:
The route should be `/file/smth`, not `/GET/smth`, GET is the method. Also, instead of `string:pathToFile` you should use `path:pathToFile`, otherwise it will stop on slashes